src/content/configuration/output.md

```diff
@@ -356,23 +356,50 @@ MySubModule.doSomething();
 
 > Default: `"var"`
 
-Configure how the library will be exposed. Any one of the following options can be used.
+Configure how the library will be exposed. Any one of the following options can be used. Please note that this option works in conjunction with the value assigned to [`output.library`](#output-library). For the following examples, it is assumed that this value is configured as `MyLibrary`.
 
-The following options are supported:
+Note that `_entry_return_` in the example code below is the value returned by the entry point. In the bundle itself, it is the output of the function that is generated by webpack from the entry point.
+
+### Exposing the library via variable
+
+These options simply assign the return value of the entry point (e.g. whatever the entry point exported) to the name provided by `output.library` at whatever scope the bundle was included at.
 
 `libraryTarget: "var"` - (default) When your library is loaded, the **return value of your entry point** will be assigned to a variable:
 
 ```javascript
 var MyLibrary = _entry_return_;
 
-// your users will use your library like:
+// your users, in a separate script, will use your library like:
```

Comment:
@metatoaster I know it's not you're original wording, but whilst we're here :)

I would add 'like the following' to 'like'.